### PR TITLE
OCM-10674 | test: automated ids:59547,62161

### PR DESCRIPTION
**Pull request by yingzhanredhat**
https://privatebin.corp.redhat.com/?98af003d3b41c1da#2manFdfRoKZz5nGRP8cSUrNdVQ9cq9rMZi6X5P7gsj74

### CODE REVIEW

File: tests/e2e/test_rosacli_upgrade.go [hunk 1]
```diff
@@ -891,6 +891,82 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+				}
+
+				By("detach managed policies from account role and update cluster")
+				arbitraryPolicyService = rosaClient.Policy
+				upgradeVersion := zStreamVersion
+				if zStreamVersion == "" {
+					upgradeVersion = yStreamVersion
+				}
+				for r, p := range rolePolicyMap {
+					_, err := arbitraryPolicyService.DetachPolicy(r, []string{p}, "--mode", "auto")
```
Comment (aaraj7) [inline tests/e2e/test_rosacli_upgrade.go:942]: There is also mention in test case to detach the policy from operator-roles and try to upgrade the cluster, Could you PTAL?

Comment (yingzhanredhat) [reply tests/e2e/test_rosacli_upgrade.go:942]: yes. This `rolePolicyMap`  contains account-roles and operator roles. It will loop every roles

Comment (aaraj7) [reply tests/e2e/test_rosacli_upgrade.go:942]: Okay Thanks for the confirmation.

File: tests/e2e/test_rosacli_upgrade.go [hunk 2]
```diff
@@ -891,6 +891,82 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+
+				for _, policyArn := range CD.OperatorIAMRoles {
+					_, operatorRoleName, err := common.ParseRoleARN(policyArn)
+					Expect(err).To(BeNil())
+					attachedPolicy, err := awsClient.ListAttachedRolePolicies(operatorRoleName)
+					Expect(err).To(BeNil())
+					rolePolicyMap[operatorRoleName] = *attachedPolicy[0].PolicyArn
+				}
+
+				By("detach managed policies from account role and update cluster")
```
Comment (aaraj7) [inline tests/e2e/test_rosacli_upgrade.go:935]:
```suggestion
				By("detach managed policies from account role and operator role and update cluster")
```

**Pull request merged**